Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Out of interest, what is this used for?

Reply:
No idea, I'll revert this. Many tracked files get randomly altered when running Mantid using VS during development. When using the git GUI you can see them and sometimes accidently include in the commit.